src/main/java/com/microsoft/azure/functions/annotation/KafkaOutput.java

```diff
@@ -36,57 +66,76 @@
     String dataType() default "";
```

Comment:
Can we add comment?

Reply:
It looks have a comment. Could you double check it?